### PR TITLE
updating environment values

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/aat/common/em/em-stitching.yaml [hunk 1]
```diff
@@ -26,6 +26,8 @@ spec:
       image: hmctspublic.azurecr.io/em/stitching:prod-464c8e27
       ingressHost: em-stitching-aat.service.core-compute-aat.internal
       environment:
+        SPRING_DATASOURCE_URL: jdbc:postgresql://em-stitching-postgres-db-aat.postgres.database.azure.com:5432/stitching
```
Comment (timja) [inline k8s/aat/common/em/em-stitching.yaml]: this should really be set in the application repo:
https://github.com/hmcts/rpa-em-stitching-api/blob/master/charts/em-stitching/values.yaml#L25

preview configuration can be overridden in values.preview.template.yaml

File: k8s/aat/common/em/em-stitching.yaml [hunk 2]
```diff
@@ -26,6 +26,8 @@ spec:
...
+        SPRING_DATASOURCE_USERNAME: annotation@em-stitching-postgres-db-aat
```
Comment (timja) [inline k8s/aat/common/em/em-stitching.yaml]: same here,
https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L9

File: k8s/aat/common/em/em-stitching.yaml (hunk 1 above)
Comment (velimd) [reply k8s/aat/common/em/em-stitching.yaml]: I set it in values.aat.template.yaml instead or is this the incorrect place?

File: k8s/aat/common/em/em-stitching.yaml (hunk 2 above)
Comment (velimd) [reply k8s/aat/common/em/em-stitching.yaml]: not sure why it would go into the draft store?

Comment (velimd) [reply k8s/aat/common/em/em-stitching.yaml]: done

File: k8s/aat/common/em/em-stitching.yaml (hunk 1 above)
Comment (velimd) [reply k8s/aat/common/em/em-stitching.yaml]: done

Comment (timja) [reply k8s/aat/common/em/em-stitching.yaml]: it should be in values.yaml

**Pull request merged**